src/plugins/saved_objects_management/public/management_section/objects_table/components/import_summary.scss

```diff
@@ -14,7 +14,3 @@
 .savedObjectsManagementImportSummary__errorCount {
   color: $euiColorDangerText;
 }
-
-.savedObjectsManagementImportSummary__icon {
-  margin-left: $euiSizeXS;
-}
```

Comment:
`.savedObjectsManagementImportSummary__icon` classname is not used anywhere and can be removed